docs/presentation/Content/DefaultPropertyCategoryOverride.md

```diff
@@ -8,26 +8,44 @@ The default property category is a category that gets assigned to properties tha
-| -------------------- | --------- | --------------------------------------------------------------------- | ------- | ---------------------------------------------------------------------------------------- |
-| *Picking attributes* |
-| `requiredSchemas`    | No        | [`RequiredSchemaSpecification[]`](../Advanced/SchemaRequirements.md)  | `[]`    | Specifications that define schema requirements for the rule to take effect.              |
-| `priority`           | No        | `number`                                                              | `1000`  | Defines the order in which presentation rules are evaluated.                             |
-| `onlyIfNotHandled`   | No        | `boolean`                                                             | `false` | Should this rule be ignored if there is already an existing rule with a higher priority. |
-| *Content Modifiers*  |
-| `specification`      | Yes       | [`PropertyCategorySpecification`](./PropertyCategorySpecification.md) |         | Specification for the custom property category                                           |
-
-## Example
-
-```JSON
-{
-  "ruleType": "DefaultPropertyCategoryOverride",
-  "requiredSchemas": [{ "name": "BisCore", "minVersion": "1.0.1" }],
-  "priority": 9999,
-  "onlyIfNotHandled": true,
-  "specification": {
-    "id": "default",
-    "label": "General Properties"
-  }
-}
+| Name                                            | Required? | Type                                                                  | Default |
+| ----------------------------------------------- | --------- | --------------------------------------------------------------------- | ------- |
+| *Picking attributes*                            |
+| [`requiredSchemas`](#attribute-requiredschemas) | No        | [`RequiredSchemaSpecification[]`](../Advanced/SchemaRequirements.md)  | `[]`    |
+| [`priority`](#attribute-priority)               | No        | `number`                                                              | `1000`  |
+| *Content Modifiers*                             |
+| [`specification`](#attribute-specification)     | Yes       | [`PropertyCategorySpecification`](./PropertyCategorySpecification.md) |         |
+
+### Attribute: `requiredSchemas`
+
+> **Default value:** `[]`
+
+Specifications that define schema requirements for the rule to take effect. See more details in [Defining ECSchema Requirements for Presentation Rules](../Advanced/SchemaRequirements.md).
+
+```ts
+[[include:Content.Customization.DefaultPropertyCategoryOverride.RequiredSchemas.Ruleset]]
+```
+
+![Example of using "required schemas" attribute](./media/defaultpropertycategoryoverride-with-requiredschemas-attribute.png)
+
+### Attribute: `priority`
+
+> **Default value:** `1000`
+
+Defines the order of the rule. Higher priority rule takes precedence over lower priority rules and, since only one rule of this type takes effect when creating content, overrides them.
```

Comment:
```suggestion
Controls rule priority. Because there can only be one default category, default category override with the highest priority value will override all other rules of the same type.
```